samples/sysbuild/with_mcuboot/README.rst

```diff
@@ -19,7 +19,7 @@ sysbuild.
 This is achieved with a sysbuild specific Kconfig configuration,
 :file:`sysbuild.conf`.
 
-The `SB_CONFIG_BOOTLOADER_MCUBOOT=y` setting in the sysbuild Kconfig file
+The ``SB_CONFIG_BOOTLOADER_MCUBOOT=y`` setting in the sysbuild Kconfig file
```

Comment:
Is this not considered a Kconfig option?

Reply:
I have to admit I have no idea how sysbuild kconfigs work. The idea behind `:kconfig:option:` is that it creates a cross-reference to the Kconfig listing. But in this case SB_CONFIG_BOOTLOADER_MCUBOOT is *not* listed there: https://docs.zephyrproject.org/latest/kconfig.html#SB_CONFIG_BOOTLOADER_MCUBOOT
We could still use the role to convey the idea that it is a kconfig option (just like we could also systematically use the role even for those local `CONFIG_SAMPLE_XYZ` option), it would just not link to anything (but still render as a literal)